packages/block-library/src/image/edit.native.js

```diff
@@ -445,6 +447,11 @@ export class ImageEdit extends Component {
 			imageDefaultSize,
 		] );
 
+		const isFeaturedImage = featuredImageId === attributes.id;
+
+		// eslint-disable-next-line no-unused-vars
+		const androidOnly = Platform.OS === 'android';
```

Comment:
Looks like the `androidOnly` const is not used anywhere, can we remove it?

Reply:
Ah, good point! I tested and there aren't any issues loading posts in iOS with this PR, so the androidOnly tag seems safe to remove. I've removed that now.